src/test/java/org/takes/rs/RsWithCookieTest.java

```diff
@@ -38,6 +38,35 @@
 public final class RsWithCookieTest {
 
     /**
+     * Carriage return constant.
+     */
+    private static final String CRLF = "\r\n";
+    /**
+     * Bar.
+     */
+    private static final String BAR = "bar";
+    /**
+     * Http ok.
+     */
+    private static final String HTTP_OK = "HTTP/1.1 200 OK";
+    /**
+     * Works.
+     */
+    private static final String WORKS = "works?";
+    /**
+     * Set cookie.
+     */
+    private static final String SET_COOKIE = "Set-Cookie: foo=works?;Path=/;";
```

Comment:
@dalifreire and this one should be a string literal inside the new private method. we can make the `works?` part a method parameter.